platform/pulp/tasking/tasks.py

```diff
@@ -225,14 +230,18 @@ def apply_async(self, *args, **kwargs):
 
         :param group_id:    The id that identifies which group of tasks a task belongs to
         :type group_id:     uuid.UUID
+        :param args:          The positional arguments to pass on to the task.
+        :type  args:          tuple
+        :param kwargs:        The keyword arguments to pass on to the task.
+        :type  kwargs:        dict
+        :param **options:     For all options accepted by apply_async please visit: http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  #NOQA
 
         :return:            An AsyncResult instance as returned by Celery's apply_async
         :rtype:             celery.result.AsyncResult
         """
-        tag_list = kwargs.pop('tags', [])
-        group_id = kwargs.pop('group_id', None)
-        async_result = super(UserFacingTask, self).apply_async(*args, **kwargs)
-        async_result.tags = tag_list
+
+        async_result = super(UserFacingTask, self).apply_async(args=args, kwargs=kwargs, **options)
+        async_result.tags = tags
```

Comment:
This line should be removed. The async_result returned does not need `tags` as an attribute. This is a holdover from the Pulp2 code which also did not use it.